zuul.d/nodeset.yaml

```diff
@@ -51,3 +51,16 @@
     groups:
       - name: computes
         nodes: []
+
+- nodeset:
+    name: centos-9-rhel-9-2-crc-extracted-2.30-3xl 
+    nodes:
+      - name: controller
+        label: cloud-centos-9-stream-tripleo
```

Comment:
Dropping the vexxhost as per https://github.com/openstack-k8s-operators/ci-framework/pull/1024